src/components/segmentedControl/index.tsx

```diff
@@ -84,6 +85,10 @@ export type SegmentedControlProps = {
   containerStyle?: StyleProp<ViewStyle>;
   style?: StyleProp<ViewStyle>;
   testID?: string;
+  /**
+   * Preset type
+   */
+  preset?: 'default' | 'form';
```

Comment:
Why not use an enum and add 'presets' statically to the SegmentedControl?

Reply:
I created a const with the presets and added it as member to the component.

Reply:
Let's talk about this point too